components/Dark-Theme/useDarkMode.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { lightTheme, darkTheme } from '@components/Dark-Theme/Themes';
+
+export const useDarkMode = () => {
+  const [theme, setTheme] = useState('light');
+  const [themeData, setThemeData] = useState(lightTheme);
+  const [mountedComponent, setMountedComponent] = useState(false);
+  const setCookie = (name, value, days) => {
+    const domain = '.realdevsquad.com';
+    const expires = new Date(Date.now() + 24 * days * 60 * 60 * 1000);
+    const cookieStr = `${name}=${value}; expires=${expires}; domain=${domain}; path=/`;
```

Comment:
`name` and `days` are always going to be **theme** and **30** , then why not use them directly over here?

Reply:
It is better to have a reusable function so that it can be used to set a different cookie as well.